tests/elf/Makefile

```diff
@@ -9,9 +9,9 @@ SOURCES = $(wildcard *.c)
 
 INCLUDES = -I$(SUBOBJDIR) -I$(INCDIR)
 
-CFLAGS = $(OEHOST_CFLAGS)
+CFLAGS = $(OEHOST_CFLAGS) $(GCOV_CFLAGS)
```

Comment:
Should protect ` $(GCOV_CFLAGS)` with `ifdef MYST_ENABLE_GCOV`

Reply:
Added this